Yosemite/Yosemite/Stores/StatsStoreV4.swift

```diff
@@ -285,6 +296,97 @@ extension StatsStoreV4 {
     }
 }
 
+
+// MARK: Convert Leaderboard into TopEarnerStats
+//
+private extension StatsStoreV4 {
+
+    /// Converts and stores a top-product `leaderboard` into a `StatsTopEarner`
+    /// Since  a `leaderboard` does not containt  the necesary product information, this method fetches the related product before starting the convertion.
+    ///
+    private func convertAndsStoreLeaderboarsIntoTopEarners(siteID: Int64,
```

Comment:
The `private` is not needed since this is a `private` extension. 🙂 

```suggestion
    func convertAndsStoreLeaderboarsIntoTopEarners(siteID: Int64,
```